charts/janssen/charts/auth-server/templates/_helpers.tpl

```diff
@@ -67,6 +67,20 @@ Create user custom defined secret envs
 {{- end }}
 {{- end }}
 
+{{/*
+Create JAVA_OPTIONS ENV for passing custom work and detailed logs
+*/}}
+{{- define "auth-server.customJavaOptions"}}
+{{ $custom := "" }}
+{{- $cnCustomJavaOptions := index .Values.global "auth-server" "cnCustomJavaOptions" }}
+{{- $custom := printf "%s" $cnCustomJavaOptions }}
+{{ $memory := .Values.resources.limits.memory | replace "Mi" "" | int -}}
+{{- $maxDirectMemory := printf "-XX:MaxDirectMemorySize=%dm" $memory -}}
+{{- $xmx := printf "-Xmx%dm" (sub $memory 300) -}}
+{{- $customJavaOptions := printf "%s %s" $maxDirectMemory $xmx -}}
+{{ $customJavaOptions | trimSuffix " " | quote }}
```

Comment:
Given `cnCustomJavaOptions: "-Drandom.attr=yes"`, the generated YAML doesn't include the value.

```
image: ghcr.io/janssenproject/jans/auth-server:1.1.1_dev
env:
  - name: CN_AUTH_JAVA_OPTIONS
    value: "-XX:MaxDirectMemorySize=2500m -Xmx2200m"      
```